docs/content/_apidocs.mdx

```diff
@@ -124,6 +126,8 @@ Dagster also provides a growing set of optional add-on libraries to integrate wi
 
 - [Papertrail](/\_apidocs/libraries/dagster-papertrail) (`dagster-papertrail`) Provides support for sending Dagster logs to Papertrail.
 
+- [Pipes](/\_apidocs/libraries/dagster-pipes) (`dagster-papertrail`) Library for inclusion in external process when using Dagster Pipes protocol.
```

Comment:
```suggestion
- [Dagster Pipes](/\_apidocs/libraries/dagster-pipes) (`dagster-pipes`) Library for inclusion in external process when using Dagster Pipes protocol.
```
lets also move it up to make it distinct from other integration libs

Reply:
👍 moved it to the top of the list